pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -624,5 +624,7 @@ self: super: builtins.intersectAttrs super {
   # need it during the build itself, too.
   cairo = addBuildTool super.cairo self.buildHaskellPackages.gtk2hs-buildtools;
   pango = disableHardening (addBuildTool super.pango self.buildHaskellPackages.gtk2hs-buildtools) ["fortify"];
-
+  
+  # Tests fails because of missing test file
+  dotenv = dontCheck super.dotenv;
```

Comment:
@cptrodolfox This looks good except for one thing: This override should actually be in the configuration-common.nix file.

Once you move it there I will go ahead and merge it in!

Reply:
Thanks, I just moved the change.

Reply:
One question though, would you mind explaining  me I little bit of the pipeline used to update Hackage package set, or pointing me to any doc. Thanks.

Reply:
@cptrodolfox I don't think there is any good documentation, but there is a video explaining it somewhat:

https://discourse.nixos.org/t/video-tutorial-how-to-re-generate-the-haskell-package-set-for-nix/4074

Also, peti does this weekly live on twitch.  If you look at past broadcasts, you can watch him do it:

https://www.twitch.tv/peti343/videos

Reply:
Thanks, I already watch the videos, will see the twitch though.